src/v/cluster/health_monitor_frontend.cc

```diff
@@ -29,12 +29,18 @@ ss::future<> health_monitor_frontend::start() {
     co_return;
 }
 
+namespace {
+void cancel_timer(ss::timer<ss::lowres_clock>& refresh_timer) {
+    if (refresh_timer.armed()) {
+        refresh_timer.cancel();
+    }
+}
+} // namespace
```

Comment:
you can call `refresh_timer.cancel()` directly -- no need for a wrapper to check if it is armed.

Reply:
oh, that is dead code from a previous change. Thanks for the flag.